run.sh

```diff
@@ -41,6 +41,11 @@ if [ ! -f /data/style/mapnik.xml ]; then
 fi
 
 if [ "$1" == "import" ]; then
+    if [ -f /var/lib/postgresql/14/main/.databaseImported ]; then 
```

Comment:
Good idea, how about just using the already existing "complete" file?

```suggestion
    if [ -f /data/database/planet-import-complete ]; then 
```
I.E.:
https://github.com/Overv/openstreetmap-tile-server/pull/360/files#diff-d31ce0453051853c17ba2a5225b3d1bfab548e095bab0967d6acfd1b3ce1b35dR130

Reply:
Or `/data/database/postgres/.databaseImported` instead, because `14` was already changed to `15` in the `master` branch.